packages/components/src/virtualized-list/utils/use-on-end-reached.test.ts

```
@@ -0,0 +1,434 @@
+		scrollHeight: 1000,
+		scrollWidth: 1000,
+		addEventListener: jest.fn((event: string, handler: Function) => {
+			listeners[event] = listeners[event] || [];
+			listeners[event].push(handler);
+		}),
+		removeEventListener: jest.fn((event: string, handler: Function) => {
+			listeners[event] = (listeners[event] || []).filter((h) => h !== handler);
+		}),
+		// Helper to trigger scroll
+		_triggerScroll() {
+			(listeners['scroll'] || []).forEach((h) => h());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)